chimedb/core/mediawiki.py

```diff
@@ -0,0 +1,65 @@
+from .orm import base_model
+from .exceptions import ValidationError
+
+import hashlib
+import peewee as pw
+
+
+class MediaWikiUser(base_model):
+    """
+    Represents a MediaWiki user.
+
+    According to https://www.mediawiki.org/wiki/Manual:User_table
+
+    Attributes
+    ----------
+    user_name : str
+        The unique username. Starts with a capital letter.
+    user_password : str
+        The password hash. See https://www.mediawiki.org/wiki/Manual:User_table#user_password
+    """
+
+    user_id = pw.IntegerField(primary_key=True)
+    user_name = pw.TextField()
+    user_password = pw.TextField()
+
+    @classmethod
+    def authenticate(cls, user, password):
```

Comment:
This needs a docstring.  It needs to _at least_ point out that the only password hashing algorithm it supports is the antiquated "B" method (which we are restricted to by the layout database web front-end).